packages/Pagination/README.md

```diff
@@ -0,0 +1,75 @@
+# @MESG-COMPONENTS/PAGINATION
+
+## Install
+
+`npm install @mesg-components/pagination` or `yarn add @mesg-components/pagination`
+
+## Properties
+
+- **total**: `Number`, **Required**. -> _length number of items_
+- **pageSize**: `Number`, **Optional**. _The default is `10`_ , -> _Size display items on page_
+- **currentPage**: `Number`, **Optional**. _The default is `1`_, -> _Number of current pagination_
+- **hideSelect**: `Number`, **Optional**. _The default is `false`_, -> _Don't display page-size select_
+
+## Event
+
+- `render-change` _this event return a current-page value and page-size value as object type_
```

Comment:
Why `render-change`? This is an event from the pagination so it should be related to the pagination not the rendering.

I would suggest having 2 events, `pageChanged` and `pageSizeChanged` and so we will have something like
```html
<pagination :total="xx" :page="xx" :page-size="xx" @page-changed="render" @page-size-changed="render" />
```

Even better we could use [sync](https://vuejs.org/v2/guide/components-custom-events.html#sync-Modifier) and have something like.
```html
<pagination :total="xx" :page.sync="xx" :page-size.sync="xx" />
```

Reply:
Fixed by using `sync`